linux/step04_all_omero.sh

```diff
@@ -25,23 +29,29 @@ fi
 if [ ! -d OMERO.server* ]; then
 	# dev branches installed via omego
 	#start-venv
-	virtualenv /home/omero/omeroenv
-	/home/omero/omeroenv/bin/pip install omego==0.6.0
+	$VIRTUALENV/bin/pip3 install omego==0.6.0
 	#end-venv
-	/home/omero/omeroenv/bin/omego download -q --ice $icevalue --branch $OMEROVER server
+	$VIRTUALENV/bin/omego download -q --ice $icevalue --branch $OMEROVER server
 fi
 
 #start-link
 ln -s OMERO.server-*/ OMERO.server
+# set OMERODIR
+export OMERODIR=OMERO.server
 #end-link
 
 #configure
-OMERO.server/bin/omero config set omero.data.dir "$OMERO_DATA_DIR"
-OMERO.server/bin/omero config set omero.db.name "$OMERO_DB_NAME"
-OMERO.server/bin/omero config set omero.db.user "$OMERO_DB_USER"
-OMERO.server/bin/omero config set omero.db.pass "$OMERO_DB_PASS"
+
+
+# Install omero-py
+pip3 install "omero-py>=5.6.dev4"
```

Comment:
This can use with unifying with the decision for the docs re: `--pre`, etc.

Reply:
This will be changed. Same for omero.server

Reply:
Which decision. I might have missed that

Reply:
There's an open question of how we suggest installing the packages across the board.

Reply:
ok so this can be finalised when we come with a decision
I understood that a decision had been made